setup.sh

```diff
@@ -25,6 +25,18 @@ pip install pycurl
 
 pip install mgmtsystem
 
+# setup test recorder
+wget https://pypi.python.org/packages/1e/8e/40c71faa24e19dab555eeb25d6c07efbc503e98b0344f0b4c3131f59947f/vnc2flv-20100207.tar.gz
+tar -zxvf vnc2flv-20100207.tar.gz
```

Comment:
ah, forgot to change that, there is a way how to install it as one-line command, URL is still there..
pip install http://pypi.python.org/packages/source/v/vnc2flv/vnc2flv-20100207.tar.gz